doc/api/date/date-formatter.md

```diff
@@ -7,7 +7,7 @@ be formatted.
 
 ### Parameters
 
-**pattern**
+**machinePattern**
```

Comment:
Please, can you revert this change? The documentation needs to be updated only when we in fact perform this change.